src/seedu/addressbook/commands/ExitCommand.java

```diff
@@ -9,6 +9,7 @@
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Exits the program.\n"
             + "Example: " + COMMAND_WORD;
+
```

Comment:
Same here. It will quickly give a bad impression to the reviewer that you have changed ExitCommand file for adding a new command called 'sort'. So try to avoid unnecessary changes. You can check the file with changes using **git status** command or IDE itself will pinpoint the updated files in a different color. You can have a quick check before commit.